pkg/component/controller/cplb_unix.go

```diff
@@ -389,7 +394,7 @@ vrrp_instance k0s-vip-{{$i}} {
 {{ if gt (len $RealServers) 0 }}
 {{ range .VirtualServers }}
 virtual_server {{ .IPAddress }} {{ $APIServerPort }} {
-    delay_loop {{ .DelayLoop }}
+    delay_loop {{ delay_loop_str .DelayLoop }}
```

Comment:
Wouldn't this "just work"?

```suggestion
    delay_loop {{ .DelayLoop.Seconds }}
```

Reply:
It does 🤦